docs/circuit_cutting/tutorials/gate_cutting_to_reduce_circuit_depth.ipynb

```diff
@@ -0,0 +1,423 @@
+      "text/plain": [
+       "<Figure size 905.294x481.6 with 1 Axes>"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "transpiled_qc.draw(\"mpl\", scale=0.4, idle_wires=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6c106db7",
+   "metadata": {},
+   "source": [
+    "### Replace distant gates with `TwoQubitQPDGate`s by specifying their indices\n",
+    "\n",
+    "`decompose_gates` will replace the gates in the specified indices with ``TwoQubitQPDGate``s and also return a list of ``QPDBasis`` instances -- one for each gate decomposition."
```

Comment:
```suggestion
    "`decompose_gates` will replace the gates in the specified indices with `TwoQubitQPDGate`s and also return a list of `QPDBasis` instances -- one for each gate decomposition."
```
since this is markdown